### PR TITLE
Arianna project-business-site

### PR DESCRIPTION
**Pull request by ariallahyar**
Here is a link to my deployed site:
https://supper-society.netlify.app/

I'm happy with how it looks on desktop and tablet, but need to work out some tweaks with the mobile view to remove accidental white space and try to adjust the "zoomed in" effect that happens after submitting the form.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,261 @@
...
+.radio-group {
+  display: flex;
+  justify-content: space-around;
+}
+
+.preference {
+  padding-right: 1rem;
+}
+
+.preference-option {
```
Comment (JaEngd) [inline code/style.css:86]: Text color inside the scroll down menu is set to blue as default in iOS.
Solve it by setting a color in the button tag (I think).

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,261 @@
...
+}
+
+form button:hover {
+  color: rgb(0,51,51);
+  font-weight: bolder;
+  background-color: white;
+  cursor: pointer;
+}
+
+.submit-form {
```
Comment (JaEngd) [inline code/style.css:115]: Love this part! 

Would've been great to have a link to like a login form or something else that takes you back to the "main view".

File: code/style.css (hunk 1 above)
Comment (ariallahyar) [reply code/style.css:86]: Thanks for pointing this out! I didn't notice it until after deploying, at which point I had no time to change it. I will look into it, and a tip to myself ns to deploy earlier to check iOS before it's too late!

File: code/style.css (hunk 2 above)
Comment (ariallahyar) [reply code/style.css:115]: Good stretch goal suggestion, I hadn't thought of that. Thanks for the idea!

File: code/index.html [hunk 3]
```diff
@@ -2,29 +2,92 @@
...
-    <!-- video or image as a header is cool :) -->
+    <!-- hero image header -->
+    <header>
+      <h1 class="header-title">Supper Society</h1>
+    </header>
+    <main>
+      <div class="container">
+        <!-- Signup form -->
+        <form id="form" action="https://httpbin.org/anything" method="post">
+          <h1 class="form-header">GET STARTED</h1>
```
Comment (JaEngd) [inline code/index.html:21]: For SEO purposes it's re

**Review by JaEngd — APPROVED** [review]
Hi Arianna,

Great work with your business site! I really enjoy the concept.

I can't find any bugs, it's responsive and works on different devices. You've met all the requirements on the Blue Level and some of the requirements on both red and black level.   

I really like the part when you submit your form and get a confirmation message. As well as the whole look which shows that you had a plan from the beginning and worked towards it. It seems that you have worked with mobile first also which is great.  

Since i'm not that good of reviewing code yet I have just made some small comments below. Let me know if you have any questions.

**Pull request closed without merging**